### PR TITLE
fix(clean): minor errors in cleaned boilerplate

### PR DESCRIPTION
**Pull request by gihrig**
templates/store.test.js - ./store is an error (should be ../store)
Requires eslint-disable-line - store.test.js is moved to tests folder by clean.

*/styles.css.hbs - stylelint-ignore is an error (should be stylelint-disable)

### CODE REVIEW

**Comment by gihrig** [comment]
Will need to be merged with #862 before it will pass CI. There is a problem with eslint and friends present in the current dependency tree (before #862).

File: internals/templates/store.test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  */
 
 import expect from 'expect';
-import configureStore from './store';
+import configureStore from '../store'; // eslint-disable-line
```
Comment (gihrig) [inline internals/templates/store.test.js]: eslint is disabled for this line as ../store is invalid when store.test.js is located in templates as a sibling of store.js. Hence ./store which satisfied eslint.

When copied into app/tests it is a level below and ./store fails to resolve.

Disabling eslint for the line seemed the most expedient solution. Really, IMO, the templates folder should mirror the app folder structure more completely.

**Comment by mxstbr** [comment]
Awesome, :shipit: when the other PR is landed! :fire: 

> IMO, the templates folder should mirror the app folder structure more completely.

I totally agree, would you mind opening an issue to track this?

**Comment by uzarubin** [comment]
I believe my PR caused this issue when I switched over to using `tests/`

**Comment by mxstbr** [comment]
Can we rebate this into master and land it?

**Comment by mxstbr** [comment]
Awesome, thanks so much for the :fire: work!

**Comment by gihrig** [comment]
Really _really_ fixed by #1414 thanks to @outdooricon! ✨

**Pull request merged**